Enhance role mapping modal based on new design

ajaypratap003 opened the pull request:
https://issues.redhat.com/browse/MGDSTRM-6396

Create role mapping:
![image](https://user-images.githubusercontent.com/11775081/149306585-cf658dc9-2fe6-4639-80f4-606349feae2e.png)

![image](https://user-images.githubusercontent.com/11775081/149306749-84f239c3-ec1a-49b7-93c4-5b0bd391ba74.png)

![image](https://user-images.githubusercontent.com/11775081/149307219-3da14dd2-b6f0-4ecd-bf1e-24a19989d701.png)


Edit role mapping:
![image](https://user-images.githubusercontent.com/11775081/149381254-0c12963a-49a3-4c5a-8df3-13378a850e93.png)

jgiardino commented on ui/src/app/pages/roles/components/modals/grantAccessModal.tsx:
```diff
@@ -320,7 +347,7 @@ export class GrantAccessModal extends PureComponent<GrantAccessModalProps, Grant
 
```
While working on updates for this, one of our content designers provided a new description to make the text more consistent with other text we're using for the button label and modal header.

Can you replace the following text:
> Manage access to resources in this Service Registry instance by assigning permissions to an account.
With this text:
> Grant access to resources in this Service Registry instance by assigning a role to an account.

Can you also replace this text:
> Manage access to resources in the Registry by assigning permissions to an account.
With this text:
> Grant access to resources in the Registry by assigning permissions to an account

ajaypratap003 replied:
done

jgiardino commented:
Your examples look great! 

There's one thing missing in the "Edit role mapping" example that you show. The selected account's current role assignment should be indicated in the radio group, similar to the third example you show for Create. For example, if the current role assignment is "Manager" then the radio button label should be "Manager (current role)".

Since I can't interact with these examples, I just want to mention that the "(current role)" label stays with that radio button if the user chooses a different radio button. I.e. it doesn't move to "Viewer" if I select that option to change the role assignment.

I also posted a comment inline about some text string changes that would be great to include as well.

ajaypratap003 commented:
> Your examples look great!
> 
> There's one thing missing in the "Edit role mapping" example that you show. The selected account's current role assignment should be indicated in the radio group, similar to the third example you show for Create. For example, if the current role assignment is "Manager" then the radio button label should be "Manager (current role)".
> 
> Since I can't interact with these examples, I just want to mention that the "(current role)" label stays with that radio button if the user chooses a different radio button. I.e. it doesn't move to "Viewer" if I select that option to change the role assignment.
> 
> I also posted a comment inline about some text string changes that would be great to include as well.

@jgiardino  Thanks for the feedback. I've updated strings as you suggested and added current role for edit. 
Current role string will not shift while changing radio options.

EricWittmann commented:
Looks really good, thanks @ajaypratap003 !
